### DISCUSSION TITLE
Parcel bundle remote/local moduel in HTML

### DISCUSSION THREAD

**Discussion by electriquo** [parcel-bundler/parcel#9454] [Q&A]
Given a HTML page as follows

```HTML
<!doctype html>
<html lang="en">
  <head>
  </head>
  <body>
    <script type="module">
      import "https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js";
    </script>
  </body>
</html>
```

How can I make Parcel bundle the script in a remote URL?

Alternatively, using NodeJS dependency manager, such as npm, the module can be downloaded locally, which changes the import URL as follows

```HTML
<!doctype html>
<html lang="en">
  <head>
  </head>
  <body>
    <script type="module">
      import "../node_modules/jquery/dist/jquery.min.js";
    </script>
  </body>
</html>
```

Using a local webserver (not the one builtin in Parcel), the HTML file opens without any issue. Still I was unable to make Parcel bundle the file, even when using [bare specifiers](https://parceljs.org/features/dependency-resolution/#bare-specifiers), and I think that when using the bare specifiers, the HTML will not load in the browser when Parcel is not use.

I am interested in a path/URL which will be resolved correctly without and with Parcel.

**Comment by mischnic** [comment]
> Using a local webserver (not the one builtin in Parcel), the HTML file opens without any issue. Still I was unable to make Parcel bundle the file, even when using [bare specifiers](https://parceljs.org/features/dependency-resolution/#bare-specifiers), and I think that when using the bare specifiers, the HTML will not load in the browser when Parcel is not use.

That should be working. Not sure why it doesn't

---

Related issue for remote imports: #4148

**Reply by electriquo** [reply]
> That should be working.

Is the example above working for you?
